resources/assets/js/components/CompaniesLeaderboard.vue

```diff
@@ -156,9 +167,19 @@ export default {
                     return company.size == filterBySize;
                 });
             }
+
+            // filter by Seacoast
+            let filterBySeacoast = this.seacoastOnly;
+            if (filterBySeacoast) {
+                companies = _.filter(companies, function(company){
+                    console.log(company.region);
```

Comment:
@shaynasylvia looks like a console log was left, can you remove and push up again? Please and thank you!